### ISSUE TITLE
Validation set hyperparameter search

### ISSUE THREAD

**Issue by evanfeinberg** [deepchem/deepchem#67]
It should be relatively straightforward to implement a train_validation set split, possibly as a carbon copy of train_test split just on the train set. This will be crucial for kosher hyperparameter searching.

**Comment by rbharath** [comment]
This was done some time ago. Going to go ahead and close this issue.
